.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+##### SUMMARY
+<!--- Describe the change below, including rationale and design decisions -->
+
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
+
+<!--- Please do not forget to include a changelog fragment:
+      https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment
+      No need to include one for docs-only or test-only PR, and for new plugins/modules PRs.
+      Read about more details in CONTRIBUTING.md.
+      -->
+
+##### ISSUE TYPE
+<!--- Pick one below and delete the rest -->
+- Bugfix Pull Request
+- Docs Pull Request
+- Feature Pull Request
+- New Module Pull Request
+- New Plugin Pull Request
+- Test Pull Request
+
+##### COMPONENT NAME
+<!--- Write the SHORT NAME of the module, plugin, task or feature below. Do not include `community.general.`! -->
```

Comment:
Is writing the component's relative path in the repo being deprecated? Or just not mentioned to simplify the comment?

Reply:
No, that also works. Also this remark applies more to the issue form, since the bot doesn't care what's filled in here for PRs. (And for issues, users usually have no idea where the corresponding file in the repo is.)

Reply:
Should be improved in 3cdbb60e0b851ddbd372ad31013670526ec6ce0c.